### PR TITLE
SX Design: add new `Image` component

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

File: src/sx-design/src/Image/Image.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    <div
+      className={styles({
+        background: true,
+        backgroundLight: theme === 'light',
+        backgroundDark: theme !== 'light',
+      })}
+      style={{ width: props.width, height: props.height }}
+    >
+      {isImageLoaded === false && props.blurhash != null ? (
+        <Blurhash hash={props.blurhash} width={props.width} height={props.height} />
```
Comment (michalsanger) [inline src/sx-design/src/Image/Image.js:44]: This fails badly if the `blurhash` is invalid. I'd recommend validation of error handling

**Review by michalsanger — APPROVED** [review]

**Comment by tbergq** [comment]
What about adding support for `webp` images, with fallback?

Doesn't have to be this PR though, just a thought. 

https://css-tricks.com/using-webp-images/

File: src/sx-design/src/Image/Image.js (hunk 1 above)
Comment (mrtnzlml) [reply src/sx-design/src/Image/Image.js:45]: Good catch! Fixed and added a test case for it. Thanks!

**Comment by mrtnzlml** [comment]
> What about adding support for `webp` images, with fallback?
> 
> Doesn't have to be this PR though, just a thought.
> 
> https://css-tricks.com/using-webp-images/

Good idea. I am happy to add it or accept a PR as long as some SX Design user wants it. YAGNI otherwise. :)

**Pull request merged**